configs/antv.json

```diff
@@ -38,44 +38,31 @@
       ]
     },
     {
-      "url": "https://antv.alipay.com/zh",
+      "url": "https://antv.vision/zh.*",
       "tags": [
         "zh"
       ]
     },
     {
-      "url": "https://antv.alipay.com/zh/.*",
+      "url": "https://*.antv.vision/zh.*",
       "tags": [
         "zh"
       ]
     },
     {
-      "url": "https://antv.alipay.com/.*/zh/.*",
-      "tags": [
-        "zh"
-      ]
-    },
-    {
-      "url": "https://antv.alipay.com/en",
-      "tags": [
-        "en"
-      ]
-    },
-    {
-      "url": "https://antv.alipay.com/en/.*",
+      "url": "https://antv.vision/en.*",
       "tags": [
         "en"
       ]
     },
     {
-      "url": "https://antv.alipay.com/.*/en/.*",
+      "url": "https://*.antv.vision/en.*",
       "tags": [
         "en"
       ]
     }
   ],
   "stop_urls": [
-    "/demo/"
```

Comment:
Please keep this stop_urls since DocSearch is only fitted to Documenation pages

Reply:
There is no `/demo/` pages on antv.vision domain now.